crates/orchestrator/src/lib.rs

```diff
@@ -89,8 +87,10 @@ struct OrchestratorState<KEY: SignatureKey> {
     /// Whether nodes should start their HotShot instances
     /// Will be set to true once all nodes post they are ready to start
     start: bool,
-    /// The total nodes that have posted they are ready to start
+    /// The nodes that have posted they are ready to start
     nodes_connected: HashSet<PeerConfig<KEY>>,
+    /// The nodes that have posted that they are ready to receive a configuration
+    nodes_identified: HashSet<(Multiaddr, PeerId)>,
```

Comment:
I think we'd want this to be a `HashMap` from `(Multiaddr, PeerId)` to `node_index`?

Reply:
Yeah that makes sense